### PR TITLE
Annotation activate compatible

### PR DESCRIPTION
**Pull request by diecui1202**
## What is the purpose of the change

support @Activate for compatible

## Verifying this change

unit tests pass

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/apache/incubator-dubbo/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist #XXX`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/incubator-dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipTests` & `mvn clean test-compile failsafe:integration-test` to make sure unit-test and integration-test pass.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/incubator-dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Comment by diecui1202** [comment]
#2144

File: dubbo-common/src/test/java/org/apache/dubbo/common/extension/ExtensionLoaderTest.java [hunk 1]
```diff
@@ -19,11 +19,7 @@
 import org.apache.dubbo.common.Constants;
 import org.apache.dubbo.common.URL;
 import org.apache.dubbo.common.extension.activate.ActivateExt1;
-import org.apache.dubbo.common.extension.activate.impl.ActivateExt1Impl1;
-import org.apache.dubbo.common.extension.activate.impl.GroupActivateExtImpl;
-import org.apache.dubbo.common.extension.activate.impl.OrderActivateExtImpl1;
-import org.apache.dubbo.common.extension.activate.impl.OrderActivateExtImpl2;
```
Comment (lovepoem) [inline dubbo-common/src/test/java/org/apache/dubbo/common/extension/ExtensionLoaderTest.java]: .* is not a good usage

Comment (diecui1202) [reply dubbo-common/src/test/java/org/apache/dubbo/common/extension/ExtensionLoaderTest.java]: OK, the IDEA does this job automatically

**Comment by diecui1202** [comment]
#2140

File: README.md [hunk 2]
```diff
@@ -148,12 +148,12 @@ Please follow the [template](https://github.com/apache/incubator-dubbo/issues/ne
 
 Please report security vulnerability to security@dubbo.incubator.apache.org (private mailing list).
 
-## Ecosystem
```
Comment (kimmking) [inline README.md]: Suggest one target with one PR.

Comment (diecui1202) [reply README.md]: Get it. I will follow it from next PR. Thanks.

**Pull request merged**